### PR TITLE
Added section about using node-prpl as a module

### PR DESCRIPTION
**Pull request by mercmobily**

### CODE REVIEW

File: pages/building-and-deploying.md [hunk 1]
```diff
@@ -60,6 +60,41 @@ npm run serve
 ### Deploying `prpl-server`
 After building, the contents of `server/` contains all the files and configuration necessary to run the app in production. The provided `server/package.json` specifies server dependencies and the start command which can be used on almost any hosting service that supports Node.js.
 
+### Using `prpl-server` as an express module
```
Comment (keanulee) [inline pages/building-and-deploying.md]: "prpl-server as a library" should be its own top section between "prpl-server (recommended)" and "Static hosting". Right now the below App Engine and Firebase steps don't follow from this section.

Table of contents needs to be updated.

Comment (mercmobily) [reply pages/building-and-deploying.md]: OK will get on to it tomorrow, going to sleep right now...

Comment (mercmobily) [reply pages/building-and-deploying.md]: Sorry I am late. Scheduled to work on it tomorrow.

Comment (mercmobily) [reply pages/building-and-deploying.md]: (...actually done now. The joy of crazy Saturday nights when you are a parent!)

**Comment by mercmobily** [comment]
Thanks for this!
Quick question: how long will it take for this to go in the doc's web site?

**Comment by frankiefu** [comment]
@mercmobily I will try to schedule one for next week.

**Comment by mercmobily** [comment]
OK!

**Pull request merged**